bin/ch-run-oci

```diff
@@ -79,27 +79,29 @@ def args_parse():
       cmd = CH_BIN + "/ch-run"
       os.execl(cmd, cmd, "--version")
 
-   bundle_ = bundle_from_cid(args_.cid)
+   bundle_ = bundle_from_cid(args_.cid, ["/tmp", "/var/tmp"])
```

Comment:
I don't believe this is the right place to define the list. Set a constant at the top and use it in `bundle_from_cid()`.